temporian/core/event_set_ops.py

```diff
@@ -2895,6 +2927,7 @@ def log(self: EventSetOrNode) -> EventSetOrNode:
 
         return log(self)
 
+
```

Comment:
extra newline here - check that your black formatting is correctly configured to run on save and using the configuration in pyproject.toml (this is already configured in `.vscode/settings.json` and should be working unless you've overriden some of the settings)